app/src/main/java/com/android/calendar/agenda/AgendaFragment.java

```diff
@@ -456,6 +467,7 @@ public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCoun
             Time t = new Time(mTimeZone);
             t.setJulianDay(mJulianDayOnTop);
             mController.setTime(t.toMillis());
+            mTime.set(t.toMillis());
```

Comment:
why is this needed?

Reply:
it's needed cause sometimes task has no time reference so I sync also mTime